kevm_pyk/src/kevm_pyk/utils.py

```diff
@@ -17,6 +21,24 @@
     splitConfigFrom,
     substitute,
 )
+from pyk.ktool import KPrint
+from pyk.ktool.kprint import build_symbol_table
+from pyk.utils import hash_str
+
+
+def KPrint_make_unparsing(_self: KPrint, extra_modules: List[KFlatModule] = []) -> KPrint:
+    modules = list(_self.definition.modules) + extra_modules
+    main_module = KFlatModule('UNPARSING', [], [KImport(_m.name) for _m in modules])
+    defn = KDefinition('UNPARSING', [main_module] + modules)
+    kprint = KPrint(str(_self.kompiled_directory))
+    kprint.definition = defn
+    kprint.symbol_table = build_symbol_table(kprint.definition, opinionated=True)
```

Comment:
We should probably change `KPrint` to take a `KDefinition` in the first place to avoid rebuilding the symbol table (and to factor out IO). What do you think?

Reply:
Issue made: https://github.com/runtimeverification/k/issues/2740